src/lib/config.js

```diff
@@ -32,6 +33,7 @@ async function updateAppsItems (config) {
           slug: app.attributes.slug,
           l10n: false,
           href: app.links.related,
+          category: CATEGORIES.includes(app.attributes.category) ? app.attributes.category : 'others',
```

Comment:
`babel-polyfill` handle this polyfill ([here](https://github.com/cozy/cozy-bar/blob/master/src/index.js#L5)) and since it was already use ([here for example](https://github.com/cozy/cozy-bar/blob/master/src/lib/config.js#L15)) I guess it's correctly handled 👍